cypress/e2e/marc/marc-authority/edit-linked-authority-record/marc-authority-edit-150-field-that-controls-marc-bib.cy.js

```diff
@@ -0,0 +1,174 @@
+    searchOption: 'Keyword',
+    authorized: 'Authorized',
+    reference: 'Reference',
+    cannotChangeCalloutMessage:
+      'Cannot change the saved MARC authority field 150 because it controls a bibliographic field(s). To change this 1XX, you must unlink all controlled bibliographic fields.',
+    cannotSaveCalloutMessage: 'Record cannot be saved without 1XX field.',
+    cannotAddCalloutMessage:
+      'Cannot add a $t to the $150 field because it controls a bibliographic field(s) that cannot control this subfield. To change this 1XX value, you must unlink all controlled bibliographic fields that cannot control $t.',
+  };
+
+  const marcFiles = [
+    {
+      marc: 'marcBibFileForC374144.mrc',
+      fileName: `testMarcFileC374144.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      instanceTitle: 'Oratore, Primitive',
+    },
+    {
+      marc: 'marcAuthFileForC374144.mrc',
+      fileName: `testMarcFileC374144.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      authorityHeading: 'Oratore',
+    },
+  ];
+
+  const createdRecordIDs = [];
+
+  before('Create test data', () => {
+    cy.loginAsAdmin({
+      path: TopMenu.dataImportPath,
+      waiter: DataImport.waitLoading,
+    }).then(() => {
+      marcFiles.forEach((marcFile) => {
+        cy.visit(TopMenu.dataImportPath);
+        DataImport.verifyUploadState();
+        DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+        JobProfiles.waitFileIsUploaded();
+        JobProfiles.waitLoadingList();
+        JobProfiles.search(marcFile.jobProfileToRun);
+        JobProfiles.runImportFile();
+        JobProfiles.waitFileIsImported(marcFile.fileName);
+        Logs.checkStatusOfJobProfile(JOB_STATUS_NAMES.COMPLETED);
+        Logs.openFileDetails(marcFile.fileName);
+        Logs.getCreatedItemsID().then((link) => {
+          createdRecordIDs.push(link.split('/')[5]);
+        });
+      });
+    });
+
+    cy.visit(TopMenu.inventoryPath).then(() => {
+      InventoryInstances.searchByTitle(createdRecordIDs[0]);
+      InventoryInstances.selectInstance();
+      InventoryInstance.editMarcBibliographicRecord();
+      InventoryInstance.verifyAndClickLinkIcon(testData.tag650);
+      InventoryInstance.verifySelectMarcAuthorityModal();
+      MarcAuthorities.switchToSearch();
+      InventoryInstance.searchResults(marcFiles[0].instanceTitle);
+      MarcAuthorities.checkFieldAndContentExistence(
+        testData.tag150,
+        testData.authority150FieldValue,
+      );
+      InventoryInstance.clickLinkButton();
+      QuickMarcEditor.verifyAfterLinkingAuthority(testData.tag650);
+      QuickMarcEditor.pressSaveAndClose();
+
+      cy.createTempUser([
+        Permissions.uiMarcAuthoritiesAuthorityRecordEdit.gui,
+        Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+        Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
```

Comment:
User should have following permissions:
"MARC Authority: View MARC authority record",
"MARC Authority: Edit MARC authority record",
"quickMARC: View, edit MARC authorities record",
"quickMARC: Can Link/unlink authority records to bib records".
Please add them all